.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "bang"]
 	path = bang
-	url = ../bang.git
+	url = https://github.com/isadorasophia/bang.git
```

Comment:
was there any reason you changed this? I previously added the relative path so the submodules could work for repositories cloned with ssh instead of https

Reply:
When the workflow ran on my fork, it would fail at the checkout step because it was looking for `https://github.com/d3adb5/{bang,gum}.git`, none of which exist.

I can drop the commit, but actions running on forks would likely run into the same scenario. What do you think?

Reply:
mmmm yeah, makes sense. does `url = ../../isadorasophia/bang` work?

Reply:
Great idea. Seems to be working just fine! Rebased on main, change is now in fa024c02.

Reply:
yay, perfect!!